docs/bugs.json

```diff
@@ -34,7 +34,7 @@
     },
     {
         "uid": "SOL-2022-2",
-        "name": "NestedCallataArrayAbiReencodingSizeValidation",
+        "name": "NestedCalldataArrayAbiReencodingSizeValidation",
```

Comment:
Note that this would have to be changed in more places. For example `bugs_by_version.json` refers to these identifiers.